dev/how-to/maintain/backup-and-restore/br.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 使用 BR 进行备份与恢复
+summary: 了解如何使用 BR 工具进行集群数据备份和恢复。
+category: reference
+---
+
+# 使用 BR 进行备份与恢复
+
+Backup & Restore（以下简称 BR）是 TiDB 分布式备份恢复的命令行工具，用于对 TiDB 集群进行数据备份和恢复。相比 Mydumper，BR 更适合大数据量的场景。本文档介绍了 BR 的工作原理、命令行描述以及详细的备份恢复用例。
```

Comment:
Per issue https://github.com/pingcap/br/issues/61, network storage is `required`, we have to make it clear in the doc.

Reply:
It's not "required" if you could combine all files from all nodes after backup, or copy the archive onto all nodes before restore.

Reply:
I tested with local storage instead of network storage:

- I combine all files from all nodes to one node, but backup failed because new TiKV cannot access the files on that one node, so this way does not work.
- Copy all the backup files to all nodes, yes, this is gonna work, but imagine this in the real scenario. BTW, the design doc said importer is used and it will upload the sst files to TiKVs.

Anyway, suggest making it clear how users should do if local storage or network storage is used.

Reply:
> BTW, the design doc said importer is used and it will upload the sst files to TiKVs.

Importer is _not_ used in this version. The design doc is outdated.